content/applications/hr/time_off.rst

```diff
@@ -465,54 +576,127 @@ Request allocation
 
 If an employee has used all their time off, or is going to run out of time off, they can request an
 allocation for additional time. Allocations can be requested in one of two ways, either from the
-:ref:`dashboard <time_off/dashboard>` or the :guilabel:`My Allocations` view.
+:ref:`Dashboard <time_off/dashboard>` or the :ref:`My Allocations <time_off/my-allocations>` view.
 
-To access the dashboard, navigate to the :menuselection:`Time Off app --> My Time Off -->
-Dashboard`. This is also the default view for the *Time Off* application.
+To access the dashboard, navigate to the :menuselection:`Time Off app --> My Time --> Dashboard`.
+This is also what is shown by default when the *Time Off* application is opened.
 
-To access :guilabel:`My Allocations`, navigate to the :menuselection:`Time Off app --> My Time Off
---> My Allocations`. This presents a list view of all the allocations for the employee.
+To access :guilabel:`My Allocations`, navigate to the :menuselection:`Time Off app --> My Time -->
+My Allocations`. This presents a list view of all the allocations for the employee.
```

Comment:
Suggestion to remove the My Allocations navigation since you referenced it above
```suggestion
```